### ISSUE TITLE
emacs-jedi-direx doesn't refresh tree (against master of direx-el)

### ISSUE THREAD

**Issue by immerrr** [tkf/emacs-jedi-direx#14]
[This commit](https://github.com/m2ym/direx-el/commit/ba8b950ba170b03acf15b5881562e9aa68814c4b) has dropped the `&key :recursive:` from `direx:item-refresh` method and refreshing fails with `wrong-number-of-arguments`.

**Comment by tkf** [comment]
I don't see any deletion in the commit you mentioned.

**Comment by immerrr** [comment]
Yeah, apparently, I've copy-pasted wrong commit id: https://github.com/m2ym/direx-el/commit/fc813e20da0f97ebcb6243484f7e5c26539caacb

**Comment by tkf** [comment]
Thanks, now I see. Since I am not using this a lot I won't fix it right now. You want to fix it?

**Comment by immerrr** [comment]
Eventually, yeah, but not right away, too. For now, I've downgraded direx via el-get.

**Comment by tkf** [comment]
OK. Le'ts see who fixes it first :-)
